src/CommunityToolkit/Xamarin.CommunityToolkit/Converters/UtcDateTimeToLocalStringConverter.shared.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Globalization;
+using Xamarin.CommunityToolkit.Extensions.Internals;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Converters
+{
+	/// <summary>
+	/// Converts the incoming value to a <see cref="string"/> local time represenation in the
+	/// datetime format provided.
+	/// </summary>
+	public class UtcDateTimeToLocalStringConverter : ValueConverterExtension, IValueConverter
+	{
+		/// <summary>
+		/// Gets or sets the datetime format value of the provided datetime or datetimeoffset for
+		/// the <see cref="UtcDateTimeToLocalStringConverter"/>. This is a bindable property.
+		/// </summary>
+		public string DateTimeFormat
+		{
+			get => (string)GetValue(DateTimeFormatProperty);
+			set => SetValue(DateTimeFormatProperty, value);
+		}
+
+		/// <summary>
+		/// Backing BindableProperty for the <see cref="DateTimeFormat"/> property.
+		/// </summary>
+		public static readonly BindableProperty DateTimeFormatProperty = BindableProperty.Create(nameof(DateTimeFormat), typeof(string), typeof(UtcDateTimeToLocalStringConverter), defaultValue: "g");
```

Comment:
please move it at the top on class

Reply:
@AndreiMisiukevich will do with the required changes shortly. I think it was the code maid extension that has reorganised the class.

Reply:
Btw, why not null by default?

Reply:
@AndreiMisiukevich Null is fine, I personally like the default so as not to throw exception and provide a general date time format. But his can be changed to null and handle this appropriately.